### DISCUSSION TITLE
HMIP-BWTH heat & cool

### DISCUSSION THREAD

**Discussion by LinusHoppe** [SukramJ/aiohomematic#1022] [Q&A]
Hey everyone,
is there any way to enable heat & cool mode for HMIP-BWTH within the same device out of the box (Channel 8)? I know the [Generic thermostat](https://www.home-assistant.io/integrations/generic_thermostat/) only allows either heat or cool, but one could use [an alternative thermostat](https://github.com/swingerman/ha-dual-smart-thermostat) to make use of that.
![grafik](https://user-images.githubusercontent.com/32911160/231695519-fa41e81b-1c30-4e3a-9923-4bef2978b5a2.png)
But is hahomematic able to support that? And if yes: How?

**Comment by SukramJ** [comment]
Add `HEATING_COOLING` to [unignore](https://github.com/danielperna84/custom_homematic#unignore-device-parameters).
